packages/vuetify/src/components/VSlider/VSlider.tsx

```diff
@@ -2,16 +2,16 @@
 import './VSlider.sass'
 
 // Components
-import { makeVInputProps, VInput } from '@/components/VInput/VInput'
-import { VLabel } from '@/components/VLabel'
 import { VSliderThumb } from './VSliderThumb'
 import { VSliderTrack } from './VSliderTrack'
+import { makeVInputProps, VInput } from '@/components/VInput/VInput'
+import { VLabel } from '@/components/VLabel'
```

Comment:
Due to how these are imported they look like they should be swapped in order. Should we just change all imports to go directly to the component and not that component's index file?

```tsx
import { VLabel } from '@/components/VLabel/VLabel'
```